bindata/network/ovn-kubernetes/007-pdb.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: ovn-raft-quorum-guard
+  namespace: openshift-ovn-kubernetes
+spec:
+  maxUnavailable: {{.OVN_MAX_UNAVAILABLE}} 
```

Comment:
Probably better to specify MinAvaiable instead.

Reply:
How come? Shouldn't change anything in terms of how K8s schedules things, or am I missing something?